### ISSUE TITLE
AttributeError: type object 'nipype' has no attribute 'omp_nthread'

### ISSUE THREAD

**Issue by mihirgandhi95** [poldracklab/mriqc#844]
Currently using a BIDS dataset.

**MRIQC ran correctly for versions 0.15.1 and 0.10.4 when run separately.** 

MRIQC version -v0.15.2 showed an error. 

**Error Details:** 
```
Purging modules
Slurm job ID:  18719253
Sun Apr 19 11:44:23 EDT 2020
Running MRIQC on sub-007
Traceback (most recent call last):
  File "/usr/local/miniconda/bin/mriqc", line 10, in <module>
    sys.exit(main())
  File "/usr/local/miniconda/lib/python3.7/site-packages/mriqc/cli/run.py", line 15, in main
    parse_args()
  File "/usr/local/miniconda/lib/python3.7/site-packages/mriqc/cli/parser.py", line 395, in parse_args
    config.nipype.omp_nthread,
AttributeError: type object 'nipype' has no attribute 'omp_nthread'
Finished running MRIQC on sub-007
Sun Apr 19 11:44:32 EDT 2020
```

MRIQC command
```
#!/bin/bash

bids_dir=/jukebox/tamir/mjgandhi/heudiconv/HEUDICONV/MRIQC_checker/mriqc_1/data/bids

singularity run --cleanenv \
    --bind $bids_dir:/home \
    --bind /usr/people \
    --bind /scratch/mjgandhi/heudiconv/work \
    /jukebox/tamir/singularity/mriqc/mriqc-v0.15.2.simg \
    --participant-label $1 \
    --correct-slice-timing --no-sub \
    --nprocs 8 -w /home/derivatives/work \
    /home /home/derivatives/mriqc participant
```

**Comment by mihirgandhi95** [comment]
To verify if the error was from the heudiconv version, I tried running three different versions of heudiconv(0.5.4, 0.6.0, 0.7.0) with the same MRIQC version (0.10.4).
All three produced correct outputs without errors. 

The second step I performed was verifying if the version of MRIQC made a difference using the most updated version of heudiconv (0.7.0).

Summary:
MRIQC version 0.15.2 gave the same error as described above every time(AttributeError: type object 'nipype' has no attribute 'omp_nthread'
).
MRIQC version 0.15.1 gave an error related to templateflow which I will check and try to solve myself.
MRIQC version 0.14.0 - Ran correctly producing the desired output.
MRIQC version 0.10.4 - Ran correctly producing the desired output.

Details:

MRIQC version 0.15.2 gave the same error as described above every time.

MRIQC version 0.15.1 gave a new error which I will check for more information on and try to solve myself:
Purging modules
Slurm job ID:  18726582
Sun Apr 19 23:23:39 EDT 2020
Running MRIQC on sub-007
2020-04-20 03:23:41,041 mriqc:IMPORTANT 
    Running MRIQC version 0.15.1:
      * BIDS dataset path: /home.
      * Output folder: /home/derivatives/mriqc.
      * Analysis levels: participant.
    
Downloading https://templateflow.s3.amazonaws.com/tpl-MNI152NLin2009cAsym/tpl-MNI152NLin2009cAsym_res-02_desc-fMRIPrep_boldref.nii.gz
Process Process-2:
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/site-packages/urllib3/connection.py", line 171, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File "/usr/local/miniconda/lib/python3.7/site-packages/urllib3/util/connection.py", line 79, in create_connection
    raise err
  File "/usr/local/miniconda/lib/python3.7/site-packages/urllib3/util/connection.py", line 69, in create_connection
    sock.connect(sa)
TimeoutError: [Errno 110] Connection timed out

MRIQC version 0.14.0:
Ran correctly producing the desired output. 

MRIQC version 0.10.4:
Ran correctly producing the desired output.
